src/node/node_info_network.h

```diff
@@ -9,6 +9,15 @@
 
 namespace ccf
 {
+  enum class EndorsementType
+  {
+    NODE,
+    NETWORK
+  };
+  DECLARE_JSON_ENUM(
+    EndorsementType,
+    {{EndorsementType::NODE, "Node"}, {EndorsementType::NETWORK, "Network"}});
```

Comment:
Looking at this again, I think `Service` would be better than `Network` as we've tried to move to `Service` for other things (e.g. service certificate).

Reply:
I'll wait for others to upvote or chime in and then make a wholesale replacement. In any case, do you mean changing only the JSON strings, or also the C++ type names?

Reply:
Only the JSON string so that it's consistent for users/operators.

Reply:
After more discussion, I'll rename it to `Service` including the C++ types.